src/backend/distributed/utils/metadata_cache.c

```diff
@@ -329,6 +335,12 @@ CitusHasBeenLoaded(void)
 		}
 
 		extensionLoaded = extensionPresent && extensionScriptExecuted;
+
+		if (extensionLoaded)
+		{
+			/* Cache pg_dist_partition's id so we can look for it to be invalidated */
+			DistPartitionRelationId();
```

Comment:
That's fine, this will help future callers. Maybe expand the comment a bit to clarify.

Minor: one-line comments should be lower-case.